WordPress/gutenberg
# Suppress block inserter focus loop

## Description


Addresses https://github.com/WordPress/gutenberg/issues/30562 (suppresses bug instead of fixing it)

Related to https://github.com/wordpress-mobile/gutenberg-mobile/pull/3681

This introduces a `setTimeout` to suppress the focus loop issue that was uncovered by the block inserter search focus issue.

## How has this been tested?




1. Run the packager from the linked Gutenberg Mobile PR: `npm run start:reset`
2. Build and run the WordPress for iOS app in Xcode
3. Open the editor and expect the bundle to load from the running packager
4. Try to reproduce the bug described in https://github.com/WordPress/gutenberg/issues/30562:
  a. With the cursor in a block that contains text, add a new block via the new block inserter search
  b. Ensure that block selection and keyboard focus don't behave unexpectedly
5. Repeat the previous step with VoiceOver on

## Checklist:
